_components/header/header.html

```diff
@@ -92,85 +86,52 @@ <h2>Header usability</h2>
+    users can use <code>Tab</code> to navigate and <code>Space</code> (or <code>Enter</code>) to open pages.
   </li>
   <li>
-    Ensure your horizontal navigation is keyboard compatible; test to make sure
-    users can use Tab to navigate and Space (or Enter) to open pages.
+    <strong>Include tab focus for all top-level navigation navigation items.</strong> This feature will
+    allow keyboard-reliant users to easily navigate interactive items.
   </li>
   <li>
-    Avoid using hover to expand dropdown lists. Hover is difficult for some
+    <strong>Avoid using hover to expand dropdown lists.</strong> Hover is difficult for some
     users and won't work on touch screens. Dropdowns should expand on click or
     with keyboard navigation.
   </li>
   <li>
-    The Design System uses semantic header and nav elements with
-    <code>role="banner"</code> and <code>role="navigation"</code> respectively.
-    <code>role="banner"</code> is your masthead.
-  </li>
-  <li>
-    You can use multiple nav elements for groups of navigation links, but you
-    should only use one <code>role="navigation"</code> for the main nav of a
-    page.
+    <strong>Add context by labelling your nav element.</strong> If your page has more than one
+    <code>nav</code> element, use the <code>aria-label</code> attribute to help assistive technology users understand the purpose of the navigation.
   </li>
   <li>
-    Use lists for your nav links — doing so enables screen readers to decipher
-    header contents.
+    <strong>Use list elements for your navigation links.</strong> This allows screen readers users to more easily
+    navigate header content.
   </li>
   <li>
-    If you’re using a logo that’s an image rather than text, make sure you
-    include alternative text for screen readers.
+    <strong>Add alt text to your logo image.</strong> If you’re using a logo that’s an image rather than text, make sure
+    you include alternative text for screen readers.
   </li>
   <li>
-    If you’re using a logo that’s text, use an <code>em</code>, not an
+    <strong>Don't use an H1 for your logo.</strong> If you’re using a logo that’s text, use an <code>em</code>, not an
     <code>h1</code>, unless it’s the homepage. Find more information here:
-    <a href="http://csswizardry.com/2010/10/your-logo-is-an-image-not-a-h1/"
-      >http://csswizardry.com/2010/10/your-logo-is-an-image-not-a-h1/.</a
-    >
+    <a
+      href="http://csswizardry.com/2010/10/your-logo-is-an-image-not-a-h1/">http://csswizardry.com/2010/10/your-logo-is-an-image-not-a-h1/.</a>
   </li>
 </ul>
 
-<h2>The “official government website” banner</h2>
-<p>
-  We recommend that all federal government websites include the “official
-  government website” banner and a logo or site name. The banner clearly
-  identifies your website as part of the United States government, while a logo or
-  site name helps visitors understand who you are and what you do.
-</p>
```

Comment:
Removed this because it didn't apply to the header component. Do we want to keep a note that suggests that adding the banner component above the header is a good idea?